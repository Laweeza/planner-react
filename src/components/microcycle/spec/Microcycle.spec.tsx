import React from 'react';
import { render, screen } from '@testing-library/react';
import Microcycle from '../lib/Microcycle';

test('renders Microcycle component', () => {
    const microcycle = render(<Microcycle />);
    expect(microcycle).toMatchSnapshot()
});
