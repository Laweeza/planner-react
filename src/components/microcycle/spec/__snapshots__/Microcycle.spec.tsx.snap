// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Microcycle component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-xaep3o-MuiPaper-root-MuiCard-root"
      >
        <div
          class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
        >
          <div
            class="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom css-18k87ye-MuiTypography-root"
          >
            Invalid Date
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-undefined css-1hbmzt3-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
            >
              <div
                aria-colcount="7"
                aria-multiselectable="false"
                aria-rowcount="0"
                class="MuiDataGrid-root MuiDataGrid-root makeStyles-dataGrid-1 MuiDataGrid-autoHeight"
                role="grid"
              >
                <div>
                  <div>
                    <div
                      class="MuiDataGrid-toolbarContainer"
                    >
                      <button
                        class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root  css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                        tabindex="0"
                        type="button"
                      >
                        Add Session
                        <span
                          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                        />
                      </button>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiDataGrid-main"
                >
                  <div
                    class="MuiDataGrid-overlay"
                    style="top: 56px;"
                  >
                    <div
                      class="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
                      role="progressbar"
                      style="width: 40px; height: 40px;"
                    >
                      <svg
                        class="MuiCircularProgress-svg"
                        viewBox="22 22 44 44"
                      >
                        <circle
                          class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
                          cx="44"
                          cy="44"
                          fill="none"
                          r="20.2"
                          stroke-width="3.6"
                        />
                      </svg>
                    </div>
                  </div>
                  <div
                    class="MuiDataGrid-columnsContainer"
                    style="min-height: 56px; max-height: 56px; line-height: 56px;"
                  >
                    <div
                      aria-rowindex="1"
                      class="MuiDataGrid-columnHeaderWrapper scroll"
                      role="row"
                      style="transform: translate3d(0px, 0px, 0px); min-width: 665px;"
                    >
                      <div
                        aria-colindex="1"
                        class="MuiDataGrid-columnHeader"
                        data-field="date"
                        role="columnheader"
                        style="width: 150px; min-width: 150px; max-width: 150px;"
                        tabindex="0"
                      >
                        <div
                          class="MuiDataGrid-columnHeaderDraggableContainer"
                          draggable="false"
                        >
                          <div
                            class="MuiDataGrid-columnHeaderTitleContainer"
                          >
                            <div
                              class="MuiDataGrid-columnHeaderTitle"
                              title=""
                            >
                              Date
                            </div>
                          </div>
                        </div>
                        <div
                          class="MuiDataGrid-columnSeparator"
                          style="min-height: 56px; opacity: 1;"
                        >
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiDataGrid-iconSeparator"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M11 19V5h2v14z"
                            />
                          </svg>
                        </div>
                      </div>
                      <div
                        aria-colindex="2"
                        class="MuiDataGrid-columnHeader"
                        data-field="name"
                        role="columnheader"
                        style="width: 130px; min-width: 130px; max-width: 130px;"
                        tabindex="-1"
                      >
                        <div
                          class="MuiDataGrid-columnHeaderDraggableContainer"
                          draggable="false"
                        >
                          <div
                            class="MuiDataGrid-columnHeaderTitleContainer"
                          >
                            <div
                              class="MuiDataGrid-columnHeaderTitle"
                              title=""
                            >
                              Name
                            </div>
                          </div>
                        </div>
                        <div
                          class="MuiDataGrid-columnSeparator"
                          style="min-height: 56px; opacity: 1;"
                        >
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiDataGrid-iconSeparator"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M11 19V5h2v14z"
                            />
                          </svg>
                        </div>
                      </div>
                      <div
                        aria-colindex="3"
                        class="MuiDataGrid-columnHeader--alignRight MuiDataGrid-columnHeader--numeric MuiDataGrid-columnHeader"
                        data-field="load"
                        role="columnheader"
                        style="width: 65px; min-width: 65px; max-width: 65px;"
                        tabindex="-1"
                      >
                        <div
                          class="MuiDataGrid-columnHeaderDraggableContainer"
                          draggable="false"
                        >
                          <div
                            class="MuiDataGrid-columnHeaderTitleContainer"
                          >
                            <div
                              class="MuiDataGrid-columnHeaderTitle"
                              title=""
                            >
                              Load
                            </div>
                          </div>
                        </div>
                        <div
                          class="MuiDataGrid-columnSeparator"
                          style="min-height: 56px; opacity: 1;"
                        >
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiDataGrid-iconSeparator"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M11 19V5h2v14z"
                            />
                          </svg>
                        </div>
                      </div>
                      <div
                        class="MuiDataGrid-cell"
                        style="min-width: 320px; max-width: 320px; line-height: 55px; min-height: 56px; max-height: 56px;"
                      />
                    </div>
                  </div>
                  <div
                    style="overflow: visible; width: 0px;"
                  >
                    <div
                      class="MuiDataGrid-windowContainer"
                      style="width: 0px; height: 136px;"
                    >
                      <div
                        class="MuiDataGrid-window"
                        style="top: 56px; overflow-y: hidden;"
                      >
                        <div
                          class="MuiDataGrid-dataContainer"
                          style="min-width: 665px; min-height: 1px;"
                        >
                          <div
                            class="MuiDataGrid-viewport"
                            style="min-width: 0; max-width: 0; max-height: 0;"
                          >
                            <div
                              class="MuiDataGrid-renderingZone"
                              style="max-height: 1px; width: 665px; transform: translate3d(0px, 0px, 0px);"
                            />
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    class="Mui-resizeTriggers"
                  >
                    <div
                      class="expand-trigger"
                    >
                      <div
                        style="width: 1px; height: 1px;"
                      />
                    </div>
                    <div
                      class="contract-trigger"
                    />
                  </div>
                </div>
                <div>
                  <div
                    class="MuiDataGrid-footerContainer"
                  >
                    <div />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-xaep3o-MuiPaper-root-MuiCard-root"
    >
      <div
        class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
      >
        <div
          class="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom css-18k87ye-MuiTypography-root"
        >
          Invalid Date
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-undefined css-1hbmzt3-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <div
              aria-colcount="7"
              aria-multiselectable="false"
              aria-rowcount="0"
              class="MuiDataGrid-root MuiDataGrid-root makeStyles-dataGrid-1 MuiDataGrid-autoHeight"
              role="grid"
            >
              <div>
                <div>
                  <div
                    class="MuiDataGrid-toolbarContainer"
                  >
                    <button
                      class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root  css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                      tabindex="0"
                      type="button"
                    >
                      Add Session
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                </div>
              </div>
              <div
                class="MuiDataGrid-main"
              >
                <div
                  class="MuiDataGrid-overlay"
                  style="top: 56px;"
                >
                  <div
                    class="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
                    role="progressbar"
                    style="width: 40px; height: 40px;"
                  >
                    <svg
                      class="MuiCircularProgress-svg"
                      viewBox="22 22 44 44"
                    >
                      <circle
                        class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
                        cx="44"
                        cy="44"
                        fill="none"
                        r="20.2"
                        stroke-width="3.6"
                      />
                    </svg>
                  </div>
                </div>
                <div
                  class="MuiDataGrid-columnsContainer"
                  style="min-height: 56px; max-height: 56px; line-height: 56px;"
                >
                  <div
                    aria-rowindex="1"
                    class="MuiDataGrid-columnHeaderWrapper scroll"
                    role="row"
                    style="transform: translate3d(0px, 0px, 0px); min-width: 665px;"
                  >
                    <div
                      aria-colindex="1"
                      class="MuiDataGrid-columnHeader"
                      data-field="date"
                      role="columnheader"
                      style="width: 150px; min-width: 150px; max-width: 150px;"
                      tabindex="0"
                    >
                      <div
                        class="MuiDataGrid-columnHeaderDraggableContainer"
                        draggable="false"
                      >
                        <div
                          class="MuiDataGrid-columnHeaderTitleContainer"
                        >
                          <div
                            class="MuiDataGrid-columnHeaderTitle"
                            title=""
                          >
                            Date
                          </div>
                        </div>
                      </div>
                      <div
                        class="MuiDataGrid-columnSeparator"
                        style="min-height: 56px; opacity: 1;"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiDataGrid-iconSeparator"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M11 19V5h2v14z"
                          />
                        </svg>
                      </div>
                    </div>
                    <div
                      aria-colindex="2"
                      class="MuiDataGrid-columnHeader"
                      data-field="name"
                      role="columnheader"
                      style="width: 130px; min-width: 130px; max-width: 130px;"
                      tabindex="-1"
                    >
                      <div
                        class="MuiDataGrid-columnHeaderDraggableContainer"
                        draggable="false"
                      >
                        <div
                          class="MuiDataGrid-columnHeaderTitleContainer"
                        >
                          <div
                            class="MuiDataGrid-columnHeaderTitle"
                            title=""
                          >
                            Name
                          </div>
                        </div>
                      </div>
                      <div
                        class="MuiDataGrid-columnSeparator"
                        style="min-height: 56px; opacity: 1;"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiDataGrid-iconSeparator"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M11 19V5h2v14z"
                          />
                        </svg>
                      </div>
                    </div>
                    <div
                      aria-colindex="3"
                      class="MuiDataGrid-columnHeader--alignRight MuiDataGrid-columnHeader--numeric MuiDataGrid-columnHeader"
                      data-field="load"
                      role="columnheader"
                      style="width: 65px; min-width: 65px; max-width: 65px;"
                      tabindex="-1"
                    >
                      <div
                        class="MuiDataGrid-columnHeaderDraggableContainer"
                        draggable="false"
                      >
                        <div
                          class="MuiDataGrid-columnHeaderTitleContainer"
                        >
                          <div
                            class="MuiDataGrid-columnHeaderTitle"
                            title=""
                          >
                            Load
                          </div>
                        </div>
                      </div>
                      <div
                        class="MuiDataGrid-columnSeparator"
                        style="min-height: 56px; opacity: 1;"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiDataGrid-iconSeparator"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M11 19V5h2v14z"
                          />
                        </svg>
                      </div>
                    </div>
                    <div
                      class="MuiDataGrid-cell"
                      style="min-width: 320px; max-width: 320px; line-height: 55px; min-height: 56px; max-height: 56px;"
                    />
                  </div>
                </div>
                <div
                  style="overflow: visible; width: 0px;"
                >
                  <div
                    class="MuiDataGrid-windowContainer"
                    style="width: 0px; height: 136px;"
                  >
                    <div
                      class="MuiDataGrid-window"
                      style="top: 56px; overflow-y: hidden;"
                    >
                      <div
                        class="MuiDataGrid-dataContainer"
                        style="min-width: 665px; min-height: 1px;"
                      >
                        <div
                          class="MuiDataGrid-viewport"
                          style="min-width: 0; max-width: 0; max-height: 0;"
                        >
                          <div
                            class="MuiDataGrid-renderingZone"
                            style="max-height: 1px; width: 665px; transform: translate3d(0px, 0px, 0px);"
                          />
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="Mui-resizeTriggers"
                >
                  <div
                    class="expand-trigger"
                  >
                    <div
                      style="width: 1px; height: 1px;"
                    />
                  </div>
                  <div
                    class="contract-trigger"
                  />
                </div>
              </div>
              <div>
                <div
                  class="MuiDataGrid-footerContainer"
                >
                  <div />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
